css/result.css

```diff
@@ -0,0 +1,28 @@
+@import url(./global.css);
+
+.score__header {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  margin: 1rem 0;
+}
+
+.question--container {
+  margin: 20px 0;
```

Comment:
Try using relative units for responsiveness

Reply:
Done.